src/test/ui/conditional-compilation/cfg_accessible.rs

```diff
@@ -8,12 +10,10 @@ mod m {
 #[cfg_accessible(m::ExistingPublic)]
 struct ExistingPublic;
 
-// FIXME: Not implemented yet.
-#[cfg_accessible(m::ExistingPrivate)] //~ ERROR not sure whether the path is accessible or not
+#[cfg_accessible(m::ExistingPrivate)]
 struct ExistingPrivate;
```

Comment:
It might be better to add something here that if the cfg evaluates to true, there is a compilation error.